entries/jQuery.validator.methods.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="jQuery.validator.methods" returns="Object">
+	<title>jQuery.validator.methods</title>
+	<desc>
+        Object holding all validation methods known to the validator.
+        This can be accessed to override individual methods, while keeping the default messages.</desc>
```

Comment:
The linebreak will collapse when rendered in the browser. If that's okay, it shouldn't be in the source. If its not okay, there should be explicit markup to render separate lines. (also mixed indent)